virtual_rainforest/abiotic/radiation.py

```diff
@@ -0,0 +1,179 @@
+"""The `abiotic.radiation` module.
+
+The radiation module calculates the radiation balance of the Virtual Rainforest.
+Top of canopy net shortwave radiation at a given location depends on
+    1. extra-terrestrial radiation (affected by the earth's orbit, time of year and
+        day, and location on the earth),
+    2. terrestrial radiation (affected by atmospheric composition and clouds),
+    3. topography (slope and aspect), and
+    4. surface albedo (vegetation type and fraction of vegetation/bare soil).
+
+Terrestial radiation can be derived from climatologies (e.g. WFDE5) or calculated here;
+the implementation is based on Davis et al., (2017). The effects of topography are taken
+into account with a factor derived from the preprocessing module.
+
+The vertical profile of net radiation below the canopy depends on the canopy
+structure, the amount of radiation absorbed by the canopy, and the longwave radiation
+emitted by canopy and surface.
+
+The radiation balance can be calculated with the radiation_balance() function. If
+terrestrial radiation is NOT provided as an input, it will be calculated internally.
+
+# the following structural components are not implemented yet
+TODO core.constants
+TODO include vertical structure (canopy layers)
+TODO include data structure (->numpy array)
+TODO include time dimension
+TODO logging, raise errors
+"""
+
+# import numpy as np
+# from core.constants import kc, kd, kfFEC, bolz # this doesn't exist yet
```

Comment:
I think this might operate as `from core.constants import CONSTANTS` and then use `CONSTANTS.k` or `CONSTANTS['k']`in code?

Or `from core.constants import CONSTANTS as C` and then `C.k` if you wanted more brevity 😄

Reply:
I will adjust as soon as we have the file in place :-)